styles/product.css

```diff
@@ -0,0 +1,43 @@
+.main-wrapper {
+  display: grid;
+  grid-template-areas:
+    "header"
+    "main";
+  grid-template-columns: auto;
+  grid-template-rows: auto 1fr;
+}
+
+.nav-wrapper {
+  grid-area: header;
+}
+
+.filter-product-wrapper {
+  grid-area: main;
+  display: grid;
+  grid-template-columns: 15rem 1fr;
+  margin-top: 3rem;
+}
+
+.filter {
+  padding: 0 1rem;
+  color: var(--heading-text-color);
+}
+.filter-head {
+  margin-bottom: 1rem;
+}
+
+.filter-section > * {
+  margin: 1rem 0;
+  list-style-type: none;
+}
+
+.filter-section > li > * {
```

Comment:
for now, it's okay. buy a small suggestion use classes for styling it will maintain your Specificity.

Reply:
Sure @ashishkatiyar2404 I will do use the classes here.

Reason Why I used css like this here.

1. I did not give any classes to list so thats why i am doing like this.
2.  I wanted every list item to look same so its like shorthand.
3. Rather adding classes to every list i prefer this method. 


But thanks for now on i will look over this piece of advice.